.github/workflows/coverage.yml

```diff
@@ -0,0 +1,64 @@
+name: Coverage QA
+
+on:
+  push:
+    branches: [ main ]
+    paths: [ '**.java', '.github/workflows/coverage.yml' ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Checkout badges branch
+        uses: actions/checkout@v2
+        with:
+          ref: badges
+          path: badges
+
+      - name: Set up JDK 1.11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
```

Comment:
I think soon in the future, we might want to use `temurin` which is the new brand name for `adoptopenjdk`